src/Analyzer.Core.BuiltInRuleTests/Tests/TA-000003/AppServicesFTPS.json

```diff
@@ -128,6 +128,55 @@
       "properties": {
         "ftpsState": "FtpsOnly"
       }
+    },
+    {
+      "type": "Microsoft.Resources/deployments",
+      "apiVersion": "2019-10-01",
+      "name": "nestedTemplate",
+      "resourceGroup": "my-rg",
```

Comment:
Can you change it to a more declarative resource name? We usually choose names that describe the test case